python/paddle/fluid/tests/unittests/prim/composite_ops/test_composite_relu_custom_vjp.py

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
it's better  to delete this if op test with prim can cover this test

Reply:
Since activation op test only test simple case, add this case can be more safe to use.